### PR TITLE
DEV-3184-new-go-exercise-concatalternate

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

File: subjects/concatalernate/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println(ConcatAlternate([]int{1,2,3},[]int{4,5,6}))
```
Comment (zainabdnaya) [inline subjects/concatalernate/README.md:32]: Could you remove space here

File: subjects/concatalernate/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+## Concat-Alternate
+
+
+### Instructions
+
+- Write a function that receives two slices of int as arguments and returns a new one with the result of alternating the values of each.
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/concatalernate/README.md]: removed

File: subjects/concatalernate/README.md (hunk 1 above)
Comment (Hamzaelkhatri) [reply subjects/concatalernate/README.md:32]: fixed

**Review by zainabdnaya — APPROVED** [review]

File: subjects/concatalternate/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+## concatalternate
+
+### Instructions
+
+Write a function `ConcatAlternate()` that receives two slices of an `int` as arguments and returns a new slice with the result of the alternated values of each slice.
+- The input slices can be of different lengths.
+- The new slice should return the elements of the larger slice first and then the elements of the smaller slice.
```
Comment (jotapero) [inline subjects/concatalternate/README.md:7]: fix: "The new slice should start with an element of the largest slice." - clearer this way

File: subjects/concatalternate/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+## concatalternate
+
+### Instructions
+
+Write a function `ConcatAlternate()` that receives two slices of an `int` as arguments and returns a new slice with the result of the alternated values of each slice.
```
Comment (jotapero) [inline subjects/concatalternate/README.md:5]: fix: (...) receives two slices of integer as arguments (...)

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/concatalternate/README.md (hunk 3 above)
Comment (tiagocollot) [reply subjects/concatalternate/README.md]: It is fixed. Thank you

**Pull request merged**